imls-backend/db/schema.sql

```diff
@@ -23,10 +30,134 @@ CREATE SCHEMA api;
 CREATE SCHEMA imlswifi;
 
 
+--
+-- Name: bin_devices_per_hour(date, text); Type: FUNCTION; Schema: api; Owner: -
+--
+
+CREATE FUNCTION api.bin_devices_per_hour(_day date, _fscs_id text) RETURNS integer[]
+    LANGUAGE plpgsql
+    AS $$
+DECLARE
+    _start TIMESTAMPTZ;
+    _end TIMESTAMPTZ;
+    _count INT;
+    _hour INT := 0;
+    _day_end INT := 24;
+    num_devices_arr INT[];
+    _timezone_offset INT;
+BEGIN
+    SELECT api.get_timezone_from_fscs_id(_fscs_id) INTO _timezone_offset;
+    _hour := _hour - _timezone_offset;
+    _day_end := _day_end - _timezone_offset;
```

Comment:
The `_hour` and `_day_end` INT variables represent the interval - so it's inclusive of `_hour` and doesn't include `_day_end` in the count because the loop on line 54 uses less than. 

So I think this is ok as is because of how `_day_end` is being used - but please let me know if I misunderstood the issue.